securedrop/alembic/versions/faac8092c123_enable_security_pragmas.py

```diff
@@ -0,0 +1,26 @@
+"""enable security pragmas
+
+Revision ID: faac8092c123
+Revises: 15ac9509fc68
+Create Date: 2018-03-31 10:44:26.533395
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'faac8092c123'
+down_revision = '15ac9509fc68'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    conn.execute(sa.text('PRAGMA secure_delete = ON'))
+    conn.execute(sa.text('PRAGMA auto_vacuum = FULL'))
```

Comment:
Good call on adding this revision - this is a good opportunity for us to get instances that installed after we established these pragmas consistent with recent installs